app/models/dinosaur.rb

```diff
@@ -9,4 +9,8 @@ class Dinosaur < ActiveRecord::Base
   validates :info_url, presence: true
   validates :user, presence: true
   validates :description, presence: true
+
+  def self.search(query)
+    where("name like ?", "%#{query.capitalize}%")
```

Comment:
You could also use `ILIKE` to deal with any casing issues, in the event that the query is all caps when coming from the user.